### DISCUSSION TITLE
Managing gh extensions with chezmoi?

### DISCUSSION THREAD

**Discussion by mathomp4** [twpayne/chezmoi#2087] [Q&A] [answered]
I'm still a newbie at chezmoi, so I'm hoping some here can teach me something new. 😄 

Namely, I recently realized that one of my computers I manage with chezmoi has some [gh extensions](https://github.com/topics/gh-extension) that another doesn't. But I figured, hey, that's what chezmoi is for!

The issue is that gh extensions are not really dotfiles but...other. Some like gh-dash (bless you @mislav) are just a big 17 MB executable and others like gh-branch are a git repo. So not really chezmoi purview

So I wondered: is there a way to use chezmoi to handle things like this, like say the using the [chezmoi and scripts](https://www.chezmoi.io/user-guide/use-scripts-to-perform-actions/#install-packages-with-scripts) bit of the code I've avoided so far in my early use?

**Comment by twpayne** [comment, marked as answer]
[Externals](https://www.chezmoi.io/user-guide/include-files-from-elsewhere/) are almost certainly the best fit here.
